contrib/pkg/broker/server/server_test.go

```diff
@@ -23,17 +23,20 @@ import (
 	"net/http/httptest"
 	"testing"
 
-	. "github.com/kubernetes-incubator/service-catalog/contrib/pkg/broker/server"
+	"github.com/kubernetes-incubator/service-catalog/contrib/pkg/broker/controller"
 	"github.com/kubernetes-incubator/service-catalog/pkg/brokerapi"
 )
 
 //
 // Test of server /v2/catalog endpoint.
 //
 
+// Make sure that Controller stub implements controller.Controller interface
+var _ controller.Controller = &Controller{}
```

Comment:
I think it also serves documentation purposes. And also makes sure that even if tests are refactored in the future and the usage as `controller.Controller` is removed/changed, this assertion is still there. My .02.